packages/terra-date-picker/src/DatePicker.module.scss

```diff
@@ -71,6 +71,7 @@
           border-color: var(--terra-date-picker-button-focus-border-color, #26a2e5);
           box-shadow: var(--terra-date-picker-button-focus-box-shadow, 0 0 1px 3px rgba(76, 178, 233, 0.5), 0 0 7px 4px rgba(76, 178, 233, 0.35));
           outline: none;
+          z-index: 2;
```

Comment:
Does this fix it if the date-picker is in a modal?

Reply:
Yes it works in modal.
<img width="831" alt="Screen Shot 2020-09-14 at 10 18 52 PM" src="https://user-images.githubusercontent.com/12869809/93114581-59e88180-f6d8-11ea-8df3-110a51ccf85d.png">